src/utils/figmaTheme/templateResolvers/test/themeRoots.js

```diff
@@ -220,6 +224,7 @@ export const compositionRoot = {
       value: {
         color: '#000000',
         borderWidth: '2',
+        spacing: '20',
```

Comment:
Is value type ever set anywhere? e.g. `px` vs `em`

Reply:
Nope it isnt ever set. its only ever numbers.